### PR TITLE
Added `shoutem configure --production` to documentation

### PR DESCRIPTION
**Pull request by Vladimir-Vdovic**
Added `shoutem configure --production` to CLI reference.
Added `shoutem configure --production` to `Publish your app` tutorial.
Added `shoutem configure --production` to `Setting up your local environment` tutorial.

### CODE REVIEW

File: docs/extensions/reference/_posts/1970-01-06-CLI.md [hunk 1]
```diff
@@ -327,17 +327,18 @@ By default, `shoutem pack` command will do `npm run build` in both `app` and `se
...
 Runs the platform's configure script to sync the local app with native changes to local extensions.
 The flag `--release` is used to switch the local app to the last published configuration on the server.
-Call the `shoutem configure` command again without the flag to switch back to the current development configuration.
+The flag `--production` is used to configure the app's bundle ID for iOS and Android to match the one provided in the Builder. It also activates the code push feature, analytics and push notifications.
 
 The command should be called when:
 - new extension is created, installed to the current app & pushed
 - changing native code of any of the extensions from the extensions/ directory
 - changing cocoapods or gradle dependencies of any of the extensions from the extensions/ directory
 - switching between published (`shoutem configure --release`) and development (`shoutem configure`) configuration
```
Comment (SoHotSoup) [inline docs/extensions/reference/_posts/1970-01-06-CLI.md]: `--release` flag, doesn't use published configuration, `--production` does. `--release` just bundles downloaded configuration, so app doesn't download it on each start. Also, `--production` flag automatically sets `release` flag.

File: docs/extensions/reference/_posts/1970-01-06-CLI.md [hunk 2]
```diff
@@ -327,17 +327,18 @@ By default, `shoutem pack` command will do `npm run build` in both `app` and `se
 
 Run this command from the extension folder.
 
-- ##### shoutem configure [--release]
+- ##### shoutem configure [--release] [--production]
 
 Runs the platform's configure script to sync the local app with native changes to local extensions.
 The flag `--release` is used to switch the local app to the last published configuration on the server.
-Call the `shoutem configure` command again without the flag to switch back to the current development configuration.
+The flag `--production` is used to configure the app's bundle ID for iOS and Android to match the one provided in the Builder. It also activates the code push feature, analytics and push notifications.
```
Comment (SoHotSoup) [inline docs/extensions/reference/_posts/1970-01-06-CLI.md]: I would add also that that bundle id (iOS) and application id (Android) could be set by changing the values of `"iosBundleId"` and `"androidApplicationId"` in the `config.json`. If they are set, one from the builder would not be respected.

File: docs/extensions/tutorials/_posts/1970-01-01-SettingLocalEnvironment.md [hunk 3]
```diff
@@ -109,13 +110,33 @@ The command should be called when:
...
-$ shoutem configure
+$ shoutem configure --release
+
+> @shoutem/mobile-app@1.1.2 configure /path/to/Restaurants
+> node scripts/configure "--release"
+...
+```
+
+This command should be called when:
+- switching between development (`shoutem configure`) and last published (`shoutem configure --release`) configuration
```
Comment (SoHotSoup) [inline docs/extensions/tutorials/_posts/1970-01-01-SettingLocalEnvironment.md]: `shoutem configure --production` not release

**Comment by Vladimir-Vdovic** [comment]
@SoHotSoup requested changes have been made.

**Pull request merged**